content/v2.0/influx-templates/create.md

```diff
@@ -1,21 +1,105 @@
 ---
-title: Create an Influx Template
+title: Create an Influx template
 description: >
-  placeholder
+  Use the InfluxDB UI and the `influx pkg export` command to create Influx templates.
 menu:
   v2_0:
     parent: Influx templates
     name: Create a template
-    identifier: Create an Influx Template
+    identifier: Create an Influx template
 weight: 101
 v2.0/tags: [influx templates]
 ---
 
-## Placeholder outline
-- Create a template
-  - `influx pkg`
-  - Package everything vs package specific things
-  - Include secrets (only keys are included, uses will still need to populate the secret)
-  - Submit a template to the InfluxDB community template repo
-  - View your package summary
-  - Validate a package
+Use the InfluxDB user interface (UI) and the `influx pkg export` command to
+create Influx templates.
+Add buckets, Telegraf configurations, tasks, and more in the InfluxDB UI and export
+them as a template.
+
+{{% note %}}
+The simplest way to create a template is to create a new organization and build
```

Comment:
why easier to create a new org and build a template than add a template in an existing org?

Reply:
Because templatable resources are scoped to an organization. So you can create an org for the sole purpose of building and maintaining a template without mixing it with other resources you don't necessarily want templated..